project_timeline/report/project_report.py

```diff
@@ -11,18 +11,33 @@ class ReportProjectTaskUser(models.Model):
     planned_date_end = fields.Datetime(readonly=True)
 
     def _select(self):
+        res = super()._select()
+        if "planned_date_end" in res:
```

Comment:
This is too broad. You may have other field `planned_date_end_custom` and do matching.

Reply:
I already make the change @pedrobaeza could you help me to review if this is the change you requested 🙏